SU-HSDO/suhumsci
# SHS-5182:  Warbler button styles/utility class button styles update

# READY FOR REVIEW

## Summary
- Update styles of `.hs-button` and `.hs-secondary-button` utility classes in warbler color scheme to ensure that they looks as expected in all cases
- Fix styles of table header links in warbler color scheme

## Need Review By (Date)
01/26

## Urgency
medium

## Steps to Test
1. Install the sociology site
2. Visit the [Traditional theme settings](https://sociology.suhumsci.loc/admin/appearance/settings/humsci_traditional) and change the color pairing to "Warbler"
3. Visit the ["PHD Placement Table" display of the "Sociology People" view config page](https://sociology.suhumsci.loc/admin/structure/views/view/custm_duplicate_of_default_people/edit/block_4), click on the  "Settings" link in the "Format" section and make the "PhD Year" column sortable.
    <img width="1178" alt="Screenshot 2024-01-24 at 17 00 58" src="https://github.com/SU-HSDO/suhumsci/assets/3496574/47b950b8-3cb8-4a78-8aa0-69e7bdf0d1a2">
4. Click on the "Content: Link to edit Content (Edit)" link in the "Fields" section, go to "Style Settings" and change the CSS class to `hs-button`
5. Save the view.
6. Visit the [PhD Placements page](https://sociology.suhumsci.loc/people/phd-placements) and confirm the following:
  a. The "PhD Year" label in the heading table is a link and its color is white
  b. The "edit" button text is white in all cases
    <img width="985" alt="Screenshot 2024-01-24 at 17 06 52" src="https://github.com/SU-HSDO/suhumsci/assets/3496574/21e82176-b59c-4022-b7a7-ff05e1332e14">
7. Edit the [view config page](https://sociology.suhumsci.loc/admin/structure/views/view/custm_duplicate_of_default_people/edit/block_4) again and change back the "Content: Link to edit Content (Edit)" field class to `hs-secondary-button`. Go back to the [PhD Placements page](https://sociology.suhumsci.loc/people/phd-placements) and confirm that the "edit" button text is white on hover.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
